src/Tests/Tests/Tests.csproj

```diff
@@ -7,7 +7,7 @@
     <NoWarn>$(NoWarn);xUnit1013</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.0-beta1-build3642" />
```

Comment:
?

Reply:
dotnet xunit 2.3.1 does not play properly with running tests from the command line - it cannot find the dotnet SDK. Reverted change back